### PR TITLE
adding italics

### PR DESCRIPTION
**Pull request by nnunn**
adding italics around "Place hold with barcode and PIN" - requested by WAT to increase visibility for alumni borrowers.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and releases in Discovery project adheres to [Semantic Versioning](http://semver
 
 ### Changed
 - create migration and models to replace status, item type, circulation rules, library and locations data from static config files and refactor to replace related constants [#1273](https://github.com/ualbertalib/discovery/issues/1273)
+- adding italics around "Place hold with barcode and PIN" to increase visability.
```
Comment (pgwillia) [inline CHANGELOG.md]: We've generally been trying to document where the change comes from.  In this case there's a little bit more information about the WAT decision in this PR.  What  do you think about adding the link?
```suggestion
- adding italics around "Place hold with barcode and PIN" to increase visability. [PR#1804](https://github.com/ualbertalib/discovery/pull/1804)
```

Comment (nnunn) [reply CHANGELOG.md]: A link to the WAT minutes?

Comment (pgwillia) [reply CHANGELOG.md]: No.  Sorry I wasn't clear.  Just a link in the Changelog to this pull request.  It's been our practice.  I've made the suggested change for you to accept if you want here: https://github.com/ualbertalib/discovery/pull/1804#discussion_r336128477

Comment (nnunn) [reply CHANGELOG.md]: right ok - I've accepted that. Thanks!

**Pull request merged**